guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,173 @@
+
+To open a channel first we will need to add some bitcoin to it. Once that is done you will be ready to make your first transaction.”
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test.png"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test@2x.png"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-mobile.jpngpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-mobile@2x.png"
+   alt-text = "Diagram showing comments from users and which section of the UI they were commenting on"
+   width = 800
+   height = 751
+%}
+
+## Here’s what we learned about educating users
+
+- Providing users with educational screenstates is a good idea. It’s always nice to keep the text as short as possible and the explanation easy. Through the usability testing, we learned that we could further simplify the text in this educational part.
+- We also learned that when providing educational screenstates we should allow users the option of going back, so that they can read the text from the previous screen.
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-2.png"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-2@2x.png"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-2-mobile.png"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-2-mobile@2x.png"
+   alt-text = "Diagram showing comments from users and which section of the UI they were commenting on"
+   width = 800
+   height = 751
+%}
+
+## In the end
+
+We had a grand time re-designing this wallet interface. We learned a lot about the mechanics behind running a design sprint. Thanks to [Hampus](https://twitter.com/hampus_s), the developer at Blixt for his input during the redesign. A big thank you to everyone from the Bitcoin Design Community who put in their time, effort and knowledge during this process. The joy of open source is that we get to join forces and do good work together, completely out in the open. If you’d like to get involved in Blixt, feel free to check out their [Github](https://github.com/hsjoberg/blixt-wallet) or join their [Telegram group](https://t.me/BlixtWallet). 
+
+
+## Resources
+
+- [Remote design sprint](https://www.figma.com/file/spI8SQHoHA174tmnEH16aS/Blixt-Design-Sprint)
+- [Interview with core developer](https://www.youtube.com/watch?v=nctibe_a4rM)
+- [Transcript of Interview](https://docs.google.com/document/d/1E6XhRxF292pCkBKxTWNmDucwVaaxddaQEiNUsUu-_5k/edit?usp=sharing)
+- [Project Overview document](https://docs.google.com/document/d/1bgd1uI0PKAn0GFZTtqLY_rzlMdYxEk-7g6KiP0QzkLU/edit#heading=h.qbzoxhysypzv)
+- [Affinity diagram](https://www.figma.com/file/yqC9MYgBxG3QNF1CzxY7uf/Blixt-Affinity-Diagram?node-id=0%3A1)
```

Comment:
This link has edit access - anyone can go in and mess up the file. Maybe change that to view access.

Reply:
Fixed